client/index.html

```diff
@@ -301,20 +301,36 @@ <h1 class="title">Settings</h1>
 								</label>
 							</div>
 							{{/if}}
+							{{#unless public}}
+							<div class="col-sm-12">
+								<h2>Push Notifications</h2>
```

Comment:
We decided to keep merging notification types and other UX/UI major improvements for later.